taskcluster/app_services_taskgraph/__init__.py

```diff
@@ -57,10 +57,12 @@ def get_decision_parameters(graph_config, parameters):
             parameters["target_tasks_method"] = "pr-skip"
         else:
             parameters["target_tasks_method"] = "pr-normal"
+    elif parameters["tasks_for"] == "cron":
+        # We don't have a great way of determining if something is a nightly or
+        # not.  But for now, we can assume all cron-based builds are nightlies.
+        parameters["nightly-build"] = True
+        parameters["target_tasks_method"] = "nightly"
```

Comment:
Nit: I wouldn't force this value here. `cron` decision task already set `target_tasks_method`.

Reply:
I'll remove the line.  Can you point to where that happens so I can understand better?

Reply:
Sure! I have some good news, the location is already in your repo, it's here: 

https://github.com/mozilla/application-services/blob/3c4c626295d098d67fe2fa34f51a5552b6d441cc/.cron.yml#L11